vm-migration/src/lib.rs

```diff
@@ -301,4 +304,8 @@ pub trait Migratable: Send + Pausable + Snapshottable + Transportable {
     fn dirty_log(&mut self) -> std::result::Result<MemoryRangeTable, MigratableError> {
         Ok(MemoryRangeTable::default())
     }
+
+    fn migration_complete(&mut self) -> std::result::Result<(), MigratableError> {
```

Comment:
nit: might be better to name it as `complete_migration` to follow the naming style here (e.g. starting with verbs)?

Reply:
I agree, I'll change it!